main.go

```diff
@@ -58,6 +60,15 @@ func sendStatus(cfg config) error {
 		"description": {getDescription(cfg.Description, cfg.Status)},
 		"context":     {cfg.Context},
 	}
+	
+	coverage := cfg.Coverage
+	
+	if _, err := strconv.ParseFloat(coverage, 32); err == nil {
+	    form.Add("coverage", coverage)
```

Comment:
Please limit it to 100.0. Maybe a warning message should be useful, if it is higher than the maximum.